src/aind_metadata_service/client.py

```diff
@@ -13,7 +13,7 @@ class StatusCodes(Enum):
     VALID_DATA = 200
     INVALID_DATA = 406
     NO_DATA_FOUND = 404
-    MULTI_STATUS = 207
+    MIXED_STATUS = 207
```

Comment:
Let's not change this name in this PR. We can create a new ticket to update the nomenclature. This will break the clients that are using this library.